app/admin/pages.rb

```diff
@@ -141,7 +142,7 @@
                       img src: "#{component.image_s3_presigned_url}", class: 'maxw-10'
                     end
                   else
-                    li "#{value}: #{component.send(key)}" if component.send(key).present?
+                    li "#{value}: #{component.send(key)}" if (component.send(key) == false || component.send(key).present?)
```

Comment:
This change I included so that boolean attrs set to `false` will show up in the top view for the pb components

Reply:
Nice! Definitely a good add on.